### PR TITLE
Shutdown GLFW after releasing GPU resources

### PR DESCRIPTION
**Pull request by hujiajie**
Fixes crash at exit

### CODE REVIEW

File: src/aquarium-optimized/dawn/ContextDawn.cpp [hunk 1]
```diff
@@ -898,7 +900,6 @@ void ContextDawn::Flush() {
 }
 
 void ContextDawn::Terminate() {
-  glfwTerminate();
```
Comment (gyagp) [inline src/aquarium-optimized/dawn/ContextDawn.cpp:901]: Do we need an empty function here?

File: src/aquarium-optimized/opengl/ContextGL.cpp [hunk 2]
```diff
@@ -447,7 +449,6 @@ void ContextGL::DoFlush(
 }
 
 void ContextGL::Terminate() {
-  glfwTerminate();
```
Comment (gyagp) [inline src/aquarium-optimized/opengl/ContextGL.cpp:450]: Same here.

File: src/aquarium-optimized/dawn/ContextDawn.cpp (hunk 1 above)
Comment (hujiajie) [reply src/aquarium-optimized/dawn/ContextDawn.cpp:901]: The only purpose is to have something symmetric with `initialize()`. But in the long term I doubt a separate `initialize()` method is needed besides the constructor.

**Comment by gyagp** [comment]
LGTM

**Pull request merged**